src/preact.d.ts

```diff
@@ -141,6 +147,7 @@ declare namespace JSX {
 		y1?:number | string;
 		y2?:number | string;
 		y?:number | string;
+		[propName: string]: any;
```

Comment:
Good lord. I just realized I put that under SVGAttributes. I dunno what happened there. I guess I was just yolo-pasting.

It is probably better suited in `HTMLAttributes`